package.json

```diff
@@ -37,7 +37,8 @@
     "typescript": "^4.1.3"
   },
   "dependencies": {
-    "axe-core": "^4.0.1"
+    "axe-core": "^4.0.1",
```

Comment:
also, I think that package should be raised to 4.2.*

Reply:
feel free to update it.